rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -62,14 +63,24 @@
       </ul>
       {% if 'genreForm' in record %}
         <ul class="list-unstyled mb-0">
-           {% for genreForm in record.genreForm %}
-             {% if 'entity' in genreForm %}
+          {% for genreForm in record.genreForm %}
+            {% if 'entity' in genreForm %}
+              {% set type, value, label = genreForm.entity | doc_entity_label(language=current_i18n.language) %}
+              {% if 'textual' == type %}
+                {% set query = 'genreForm.entity.authorized_access_point_' ~ current_i18n.language ~ ':"' ~ value ~ '"' %}
```

Comment:
for my information : what is the purpose/usage of the `~` characters ?

Reply:
The `~` sign is used to concatenate values.